multinet/db.py

```diff
@@ -202,20 +210,21 @@ def edge_table(fields: Sequence[str]) -> bool:
         return "_from" in fields and "_to" in fields
```

Comment:
Looks like this and line above are no longer being used now that you use `properties()`

Reply:
Good catch